packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.Props.ts

```diff
@@ -56,9 +73,9 @@ export interface IActivityItemProps extends React.AllHTMLAttributes<HTMLElement>
   styles?: IActivityItemStyles;
 
   /**
-   * Text shown as a timestamp on this activity. If not included, no timestamp is shown.
+   * Element shown as a timestamp on this activity. If not included, no timestamp is shown.
    */
-  timeStamp?: string;
+  timeStamp?: React.ReactNode[] | React.ReactNode;
```

Comment:
to be a non breaking change this would be `string | React.ReactNode | React.ReactNode[]`